src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs

```diff
@@ -2225,4 +2225,14 @@ private class IntAsClass
             public int Value;
         }
     }
+
+    // Ensure that all common tests are still passing when the AcceptLineFeedAsLineTerminator quirk mode is enabled.
+    public class RequestTestsQuirkMode : RequestTests
```

Comment:
Remove

Reply:
The goal is to run the current benchmark with the switch on. Isn't that necessary to ensure that it's not breaking anything, or is there a better way to do it?

Reply:
We can't check in anything that sets AppContext switches, they break concurrent tests. We need to control the behavior via KestrelServerOptions.

Reply:
Updated, not removed.